src/script/script-attributes.js

```diff
@@ -33,14 +34,31 @@ var rawToValue = function (app, args, value, old) {
             }
             return null;
         case 'json':
-            if (typeof value === 'object') {
-                return value;
+            var result = {};
+
+            if (!value || typeof value !== 'object') {
```

Comment:
could move this inside the block starting line 43

Reply:
Done